associations.rb

```diff
@@ -0,0 +1,52 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+  end
+end
```

Comment:
- [ ] Kindly, as requested in the project requirements on the second point [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany/belongs%2Dto%20relationship%20between%20Classroom%20and%20Student.%20The%20following%20should%20be%20implemented%3A), make sure the relation `has-many` `belongs to` relationship is implemented. in such as way that `when adding a student to a classroom it also sets the classroom for the student` and `when setting the classroom for a student it also adds it to the classrooms' students.`
- [ ] Also the `Student`class is supposed to inherit from the `Person`

Hint: YOu may consider saying something like ;
```rb
class Student < Person
  attr_reader :classroom
  .
  .
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end

```